### DISCUSSION TITLE
Checksum errors

### DISCUSSION THREAD

**Discussion by Inflatablemango** [sanni/cartreader#299] [Need help]
hi everyone so im getting check sum error everytime i try to dump a rom from any cart no matter what the console  i can access the saves fine but the roms load the name of the game up then when it gets to the checking checksum section it just fails and says checksum error 222a ? any ideas ![IMG_20211204_162629.jpg](https://user-images.githubusercontent.com/95540049/144717028-2887ea99-65ec-4515-8d09-1ae22aff920d.jpg)![IMG_20211204_162643.jpg](https://user-images.githubusercontent.com/95540049/144717034-76f45a88-b768-46b6-8fa5-2dd00bbe024c.jpg)

**Comment by sanni** [comment]
Open the dump in an hex editor and compare it to a known good rom. Usually from the differences you can spot where the error is and reflow the solder of those pins.
See also this more detailed explanation: https://github.com/sanni/cartreader/discussions/297#discussioncomment-1748479

**Reply by Dakkaron** [reply]
If all consoles have the problem, either you have a short somewhere or your Arduino isn't soldered correctly. Try disconnecting the Arduino and check all pins with a continuity tester.
